New package: ClusterUtils v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: ClusterUtils
- Repository: https://github.com/HespanhaPublic/ClusterUtils.jl
- Created by: @hespanha
- Version: v1.0.0
- Commit: 1b8b8ccb58e3594f1c4a70ca7a0df271805ba98d
- Reviewed by: @hespanha
- Reference: https://github.com/HespanhaPublic/ClusterUtils.jl/commit/1b8b8ccb58e3594f1c4a70ca7a0df271805ba98d#commitcomment-157967929
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd64f24cad4bca2e1332e446f46ea1ba964209f068200868e04707bfe038d347f2f200918bbef42a63ab3344c0227722773c2a1c063cd7aeefb7e696c65dcf774c813c993adf4afda94d58051c29490574629973f1b915fe1390cd366aac85eda9236a5e4226eb4e2b7727ac1bb93af1999197bea7be649a8c781400d40ee8f0b1b5bfbb967b272c2de38f5d1596f82c4d92e2d823d22bc1736a2e96748190fb0dc6e30f2bacc02ea222c3ec7de12f0dec5 -->

Moelf commented:
why not ClusterManager.jl?

[noblock]

hespanha commented:
@Moelf Thanks for the question.

A portion of this package does build upon [ClusterManagers](https://github.com/JuliaParallel/ClusterManagers.jl/tree/master), but uses ssh-tunneling to enable communication between the cluster workers and a local computer that does not have a public IP addressed (e.g., in a private network with NAT). This was the only way I found to use Distributed.jl from my computer at hoime, with workers running on a Slurm cluster.

I've added this explnation to the README.md file (but I'll need to re-register because my public repository was missing one file).

[noblock]

JuliaRegistrator committed:
New package: ClusterUtils v1.0.0

UUID: 651d1c28-5f8d-4b26-91d9-468bf8b4a262
Repo: https://github.com/HespanhaPublic/ClusterUtils.jl.git
Tree: 8864f787b731fe18457684e618deca64456771e6

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

hespanha commented:
Sorry.
Yes, i did mean [noblock]

On Tue, May 27, 2025 at 3:43 AM Michael Goerz ***@***.***>
wrote:

> *goerz* left a comment (JuliaRegistries/General#131670)
> <https://github.com/JuliaRegistries/General/pull/131670#issuecomment-2912048097>
>
> [block]
>
> Do you mean [noblock]?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaRegistries/General/pull/131670#issuecomment-2912048097>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC3N7NM66JUBDGPTIVBIHY33AQ6VLAVCNFSM6AAAAAB53F46CWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSMJSGA2DQMBZG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
